src/contexts/sendSearchRequest.ts

```diff
@@ -47,7 +49,7 @@ export const sendSearchRequest = async ({
       {
         query: query_str,
         start: 0,
-        numResults: 10,
+        numResults: rerank ? 50 : 10,
```

Comment:
Using 50 as the default number of results, but I'm not 100% sure that's an optimal number. We may want to expose that as another configurable parameter.